src/api.coffee

```diff
@@ -7,7 +7,7 @@
 
 $ = require 'jquery'
 _ = require 'underscore'
-
+CSRF_Token = document.head.querySelector('meta[name=csrf-token]')?.getAttribute("content")
```

Comment:
Maybe, but if we do that then we'll also have to modify the dev server to set a dummy meta tag.  My thoughts were that the validation of the tag's presence and accuracy would be performed by the server when requests are performed.

I'd suggest that we split the difference and `console.warn` a "CRSF Token not detected" message.